docs/user/azure/install_upi.md

```diff
@@ -288,11 +289,17 @@ Create the deployment using the `az` client:
 
 ```sh
 export VHD_BLOB_URL=`az storage blob url --account-name ${CLUSTER_NAME}sa --account-key $ACCOUNT_KEY -c vhd -n "rhcos.vhd" -o tsv`
+export RELEASE_VERSION=`openshift-install coreos print-stream-json | jq -r --arg arch "$OCP_ARCH" '.architectures[$arch]."rhel-coreos-extensions"."azure-disk".release'`
+export STORAGE_ACCOUNT_ID=`az storage account show -g ${RESOURCE_GROUP} --name ${CLUSTER_NAME}sa --query id -o tsv`
+export AZ_ARCH=`echo $OCP_ARCH | sed 's/x86_64/x64/;s/aarch64/Arm64/'`
 
 az deployment group create -g $RESOURCE_GROUP \
   --template-file "02_storage.json" \
   --parameters vhdBlobURL="$VHD_BLOB_URL" \
-  --parameters baseName="$INFRA_ID"
+  --parameters baseName="$INFRA_ID" \
+  --parameters imageRelease="${RELEASE_VERSION::15}" \
```

Comment:
Why have to limit to 15 length?

Reply:
That's a good question. It's because the RHCOS version is in the format "412.86.202209302317-0" and the image version [1]:
 * Can only have numbers and periods on it
 * Each segment has to fit into a `int32`

So we use the RHCOS version up to the day (in the sample above it would be "412.86.20220930").

[1] https://learn.microsoft.com/en-us/azure/templates/microsoft.compute/2021-10-01/galleries/images/versions?pivots=deployment-language-arm-template#galleriesimagesversions-1

Reply:
How about introduce a function to convert the string inside ARM template, but only ask users to input the whole release version string gotten from installer. That will be more straight-forward.

Reply:
That's what I did initially but then changed based on https://github.com/openshift/installer/pull/6684#discussion_r1043872859

Reply:
@Prashanth684 what do you think?

Reply:
After some discussions in Slack, we've decided to drop this parameter and use a static value instead. Asking the user to pass in the value creates some work for them with honestly very little value that we're aware of. This was a consequence of us not knowing until recently you can just use `latest` when referring to an image.

Reply:
@r4f4 though I prefer the idea, will that become a problem when users is scaling up via machine-api in a UPI install?

Reply:
@jianlinliu It shouldn't be since we just use `latest` as the version for MAO: https://github.com/openshift/installer/blob/master/pkg/asset/machines/azure/machines.go#L120